Gems/Atom/Feature/Common/Assets/Passes/DownsampleSinglePassMipChain.pass

```diff
@@ -0,0 +1,90 @@
+{
+    "Type": "JsonSerialization",
+    "Version": 1,
+    "ClassName": "PassAsset",
+    "ClassData": {
+        "PassTemplate": {
+            "Name": "DownsampleSinglePassMipChainTemplate",
+            "PassClass": "DownsampleSinglePassMipChainPass",
+            "PassData": {
+                "$type": "ComputePassData",
+                "Make Fullscreen Pass": false,
+                "ShaderAsset": {
+                    "FilePath": "Shaders/PostProcessing/DownsampleSinglePassMinAvgMax.shader" 
+                }
+            },
+            "Slots": [
+                {
+                    "Name": "MipChainInputOutput",
+                    "SlotType": "InputOutput",
+                    "ShaderInputName": "m_imageDestination",
+                    "ScopeAttachmentUsage": "Shader",
+                    "ImageViewDesc": {
+                        "MipSliceMin": 0,
+                        "MipSliceMax": 12
+                    }
+                },
+                {
+                    "Name": "Mip6",
+                    "SlotType": "InputOutput",
+                    "ShaderInputName": "m_imageDestination6",
+                    "ScopeAttachmentUsage": "Shader"
+                },
+                {
+                    "Name": "GlobalAtomic",
+                    "SlotType": "InputOutput",
+                    "ShaderInputName": "m_globalAtomic",
+                    "ScopeAttachmentUsage": "Shader",
+                    "BufferViewDesc": {
+                        "m_elementSize": 4,
+                        "m_elementCount": 1
+                    }
+                }
+            ],
+            "ImageAttachments": [
+                {
+                    "Name": "Mip6",
```

Comment:
Does mip6 need to be it's own dedicated image attachment? I assume that from the final output, you would be able to create a separate `ImageView` that binds the 6th mip level directly in C++

Reply:
Thanks.  I tried to bind the 6-th mip image view to m_imageDestination6 slot and it works.  I can delete this image generation.